processor/src/trace/virtual_bus/multilinear/mod.rs

```diff
@@ -85,6 +85,7 @@ impl<E: FieldElement> MultiLinearPoly<E> {
     /// Computes f(y_1, ..., y_{ν - 1}, r_0) using the linear interpolation formula
     /// (1 - r_0) * f(y_1, ..., y_{ν - 1}, 0) + r_0 * f(y_1, ..., y_{ν - 1}, 1) and assigns
     /// the resulting multi-linear, defined over a domain of half the size, to `self`.
+    #[allow(dead_code)]
```

Comment:
I think we can remove `bind_right()`, since we won't need it?

And potentially rename `bind_left()` to `bind_least_significant_variable()` (if we adopt my other renaming suggestion of `project_lower_variable()`)